hardhat.config.ts

```diff
@@ -12,7 +12,9 @@ import { HardhatUserConfig } from "hardhat/config";
 
 const { task } = require("hardhat/config");
 const testMnemonic = 'inhale wood champion certain immense wash pepper enact enrich infant purse maid'
-const INFURA_KEY = process.env.INFURA_API_KEY;
+const RINKEBY_URL = process.env.RINKEBY_URL;
+const ROPSTEN_URL = process.env.ROPSTEN_URL;
+const MAINNET_URL = process.env.MAINNET_URL;
```

Comment:
These 3 Environment variables should be added in the `ci.yaml`, `code-coverage-check.yaml`.
Without these, the CI will break
e.g. 
```
jobs:
  build:
    env:
      RINKEBY_URL: https://provider_url/project_key
      ROPSTEN_URL: https://provider_url/project_key
      MAINNET_URL: https://provider_url/project_key
```

Reply:
Yes, I know. Working on it. I can't add it until a secret is created.

Reply:
Resolving this conversation as the Github actions are working fine now.